x-pack/plugins/observability_solution/infra/public/pages/metrics/inventory_view/components/waffle/node.tsx

```diff
@@ -60,55 +58,47 @@ export const Node = ({
 
   const showContainerAssetDetailPage = nodeType === 'container' && isContainerAssetViewEnabled;
 
-  const toggleAssetPopover = () => {
+  const toggleAssetPopover = useCallback(() => {
     if (nodeType === 'host' || showContainerAssetDetailPage) {
       setFlyoutUrlState({ detailsItemId: node.id, assetType: nodeType });
     } else {
       togglePopover();
     }
-  };
-
-  const nodeSquare = (
-    <NodeSquare
-      squareSize={squareSize}
-      togglePopover={toggleAssetPopover}
-      showToolTip={showToolTip}
-      hideToolTip={hideToolTip}
-      color={color}
-      nodeName={node.name}
-      value={value}
-      showBorder={detailsItemId === node.id || isPopoverOpen}
-    />
-  );
+  }, [node.id, nodeType, setFlyoutUrlState, showContainerAssetDetailPage, togglePopover]);
 
   return (
     <>
-      {isPopoverOpen ? (
-        <EuiPopover
-          button={nodeSquare}
-          isOpen={isPopoverOpen}
-          closePopover={closePopover}
-          anchorPosition="downCenter"
-          style={{ height: squareSize }}
-        >
-          <NodeContextMenu
-            node={node}
-            nodeType={nodeType}
-            options={options}
-            currentTime={currentTime}
+      <EuiToolTip
+        delay="regular"
+        position="right"
+        content={<ConditionalToolTip currentTime={currentTime} node={node} nodeType={nodeType} />}
+      >
+        <div>
```

Comment:
Q: Is this `div` needed? Should we remove it?

Reply:
The EUI tooltip is not displayed when navigating with the keyboard without this wrapper. I have not found a way to solve this problem

Reply:
I am also not sure maybe we can check with @elastic/eui-team. It's not blocking for the PR.